rpcs3/Gui/MainFrame.cpp

```diff
@@ -73,7 +73,7 @@ MainFrame::MainFrame()
 	, m_sys_menu_opened(false)
 {
 
-	SetLabel(wxString::Format(_PRGNAME_ " " _PRGVER_ " " RPCS3_GIT_VERSION));
+	SetLabel(wxString::Format(_PRGNAME_ " " RPCS3_GIT_VERSION));

```

Comment:
It might be actually useful to make the RPCS3_GIT_VERSION be the commit hash and _PRGVER_ stay here.
Ie. "RPCS3 0.0.0.6-7bbd245"